### PR TITLE
add installation doc without vagrant

### PR DESCRIPTION
**Pull request by nicolasThal**

### CODE REVIEW

File: en/developer_guide/install.rst [hunk 1]
```diff
@@ -1,149 +1,117 @@
-Download OpenOrchestra
-======================
+Install OpenOrchestra on your computer
+======================================
 
 Open Orchestra is composed of two projects : *open-orchestra* which is the CMS Backoffice and *open-orchestra-front-demo* which
```
Comment (ngilain) [inline en/developer_guide/install.rst:4]: Must we talk about projects or products ? @nbouquet : how do we communicate about Open Orchestra ?

File: en/developer_guide/install.rst [hunk 2]
```diff
@@ -1,149 +1,117 @@
...
+.. code-block:: bash
 
-When you launch the box, it will take some time to :
+    $ ./composer.phar create-project open-orchestra/open-orchestra path/to/your/folder -s dev
+    $ ./composer.phar create-project open-orchestra/open-orchestra-front-demo path/to/your/folder -s dev
 
-* Import the base box
-* Launch it
-* Run all the provisionning scripts
+Clone the provisioning repository :
```
Comment (ngilain) [inline en/developer_guide/install.rst]: Is this still required without vagrant ?

File: en/developer_guide/install_with_vagrant.rst [hunk 3]
```diff
@@ -0,0 +1,162 @@
+Install OpenOrchestra with vagrant
+==================================
+
+Open Orchestra is composed of two projects : *open-orchestra* which is the CMS Backoffice and *open-orchestra-front-demo* which
+is the Frontoffice part that will display the sites and pages created in the CMS.
+
+Requirements
+------------
+
+To install Open Orchestra you need to have installed the following software on your server:
```
Comment (ngilain) [inline en/developer_guide/install_with_vagrant.rst]: softwares

File: en/developer_guide/install_with_vagrant.rst [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+    $ source ./hacking/env-setup
+
+Go on the project page for more inforation : http://www.ansible.com
+
+Download Composer
+-----------------
+
+Composer is the package manager used by modern PHP applications.
+
+To install composer with curl:
```
Comment (itkg-ngilain) [inline en/developer_guide/install_with_vagrant.rst]: You should mention where to launch this command (ie on the project root folder)

**Comment by nicolasThal** [comment]
@itkg-ngilain doc updated

File: en/developer_guide/install_with_vagrant.rst [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+    192.168.33.10   media.openorchestra.dev
+
+Launch the box
+--------------
+
+When you launch the box, it will take some time to :
+
+* Import the base box
+* Launch it
+* Run all the provisionning scripts
```
Comment (ngilain) [inline en/developer_guide/install_with_vagrant.rst:121]: It should be mentionned here where to run this command (ie the project folder)

**Comment by nicolasThal** [comment]
@itkg-ngilain doc updated

File: en/developer_guide/install.rst [hunk 6]
```diff
@@ -1,149 +1,111 @@
...
 
 .. code-block:: bash
 
-    $ aptitude install virtualbox
-    $ wget https://dl.bintray.com/mitchellh/vagrant/vagrant_1.6.3_x86_64.deb
-    $ dpkg -i vagrant_1.6.3_x86_64.deb
-
-Install ansible
----------------
+    $ sudo aptitude install php5-cli php5-dev php-pear php5-curl
```
Comment (ngilain) [inline en/developer_guide/install.rst]: As https://github.com/open-orchestra/open-orchestra-docs/pull/52/files shows, there are some other non-standard extensions required by Orchestra, maybe the doc could talk about them (Imagick, Apc, Mbstring)

File: en/developer_guide/install.rst [hunk 7]
```diff
@@ -1,149 +1,111 @@
...
 Install the assets
 ------------------
 
 We are using npm to manage some server side javascript librairies and bower to manage the client side librairies
 
-Connect to the vagrant box using `vagrant ssh`, then go in the project directory inside the box
-
-.. code-block:: bash
-
-    $ cd /var/www/openorchestra
```
Comment (itkg-ngilain) [inline en/developer_guide/install.rst]: A line like this saying where to run the grunt command and the followings lacks

**Comment by nicolasThal** [comment]
@itkg-ngilain doc updated

File: en/developer_guide/install.rst [hunk 8]
```diff
@@ -1,149 +1,111 @@
...
+If you don't have curl installed, you can also download it with php:
 
-.. code-block:: text
+.. code-block:: bash
 
-    192.168.33.10   admin.openorchestra.dev
-    192.168.33.10   front.openorchestra.dev
-    192.168.33.10   demo.openorchestra.dev
-    192.168.33.10   media.openorchestra.dev
+    $ php -r "readfile('https://getcomposer.org/installer');" | php
```
Comment (itkg-ngilain) [inline en/developer_guide/install.rst]: In that doc, there's a link between where you run that command and the directory where you install Orchestra (as there's no copy of composer.phar) Maybe the doc could say where to run that command in preview of the orchestra install

File: en/developer_guide/install_with_vagrant.rst [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+----------------------------
+
+In the ``/etc/hosts`` file of your computer add the following lines :
+
+.. code-block:: text
+
+    192.168.33.10   admin.openorchestra.dev
+    192.168.33.10   front.openorchestra.dev
+    192.168.33.10   demo.openorchestra.dev
+    192.168.33.10   media.openorchestra.dev
```
Comment (ngilain) [inline en/developer_guide/install_with_vagrant.rst]: I'm not sure this domain name is still used

**Comment by ngilain** [comment]
:+1: with the remove of media.open.orchestra.dev binding (if required)

**Pull request merged**